### DISCUSSION TITLE
Which wallet connect version does rainbowkit use?

### DISCUSSION THREAD

**Discussion by saitejas-cypherd** [rainbow-me/rainbowkit#927] [Q&A]
I would like to know which wallet connect version does rainbowkit use - 1.0 or 2.0?

Will it support for wallets which use 1.0?

**Comment by mcdaded** [comment]
Looks like it uses WalletConnect from wagmi https://github.com/rainbow-me/rainbowkit/blob/ecd72094c6e5ef56a493ccda45f35cd4631b1d9c/packages/rainbowkit/src/wallets/connectorsForWallets.ts 

Seems that Wagmi is using 1.0 (at least for now)
